@dungeonmaster18, thanks for making your first contribution! Much appreciated.

I don't think this is quite the right fix, though! The error I'm seeing from `golint` is:

```
docs/generated/sql/doc.go:15:1: package comment should be of the form "Package sql ..."
```

Your proposed fix makes the comment in question no longer a "package comment" by introducing a blank line between the comment and the `package sql` declaration, but that comment _is_ supposed to be a package comment. I propose we instead adjust the wording to follow `golint`'s advice. Perhaps something like:

```diff
diff --git a/docs/generated/sql/doc.go b/docs/generated/sql/doc.go
index 240825479..4260ca368 100644
--- a/docs/generated/sql/doc.go
+++ b/docs/generated/sql/doc.go
@@ -13,8 +13,8 @@
 // permissions and limitations under the License.
 
 /*
-This dummy package contains documentation generated from sources in
-`pkg/sql/parser` using `docgen`.
+Package sql is a dummy package that contains documentation generated from
+sources in `pkg/sql/parser` using `docgen`.
 
 These files are copied to the `docs` repository where they are included in pages
 that document SQL. This happens periodically to ensure they stay up-to-date, for
```

Would you mind to update the PR?

--- a/docs/generated/sql/doc.go
+++ b/docs/generated/sql/doc.go
@@ -13,8 +13,8 @@
 // permissions and limitations under the License.
 
 /*
-This dummy package contains documentation generated from sources in
-`pkg/sql/parser` using `docgen`.
+Package sql is a dummy package that contains documentation generated from
+sources in `pkg/sql/parser` using `docgen`.
 
 These files are copied to the `docs` repository where they are included in pages
 that document SQL. This happens periodically to ensure they stay up-to-date, for

